### ISSUE TITLE
add "secondary icon" property

### ISSUE THREAD

**Issue by tyrasd** [ideditor/schema-builder#135]
Often, especially for relatively specific presets, it is hard to find a suitable icon which matches the preset perfectly. In many cases, these are presets which are a specific subtype of a more generic preset, let's take for example charging station for electric bicycles (from https://github.com/openstreetmap/id-tagging-schema/issues/1085): This could use the generic icon for charging stations as its main icon, but refine it by adding a secondary qualifier icon which represents the sub type. Here are some quick mockups of how it could look like in iD:

<img width="320" src="https://github.com/ideditor/schema-builder/assets/1927298/0459de0c-49a4-4404-befb-599fedc0e488"/>
<br>
<img width="320" src="https://github.com/ideditor/schema-builder/assets/1927298/d03a5922-96bd-4c8d-9cac-f9f085231aac"/>

**Comment by 1ec5** [comment]
Do you envision name-suggestion-index presets also putting a badge on the main icon?

**Comment by tordans** [comment]
There are also the map pins to consider.

**Comment by tyrasd** [comment]
> Do you envision name-suggestion-index presets also putting a badge on the main icon?

Yeah, this is a closely related consideration. I think it would make sense to show the brand icon similar to the secondary icons depicted above: this would make the type of map feature recognizable even if a user does not know a particular brand or when a brand operates different kind of facilities.

**Comment by tyrasd** [comment]
> There are also the map pins to consider.

Yes. But given that these have to be rendered pretty small, it is likely that the pins will continue to use just the primary icons.

**Comment by 1ec5** [comment]
> I think it would make sense to show the brand icon similar to the secondary icons depicted above: this would make the type of map feature recognizable even if a user does not know a particular brand or when a brand operates different kind of facilities.

I could see this being useful particularly for NSI’s `operator` tree. After all, one can reasonably say, “This is a McDonald’s,” but not necessarily, “This is a National Park Service,” just based on who operates the POI. But one downside is that many brand icons are already difficult enough to recognize at the size used for primary icons, let alone when downsized to fit your mocked-up secondary icon box.

**Comment by tyrasd** [comment]
some additional cases of presets which could maybe make use of this feature:

* flush toilets (maybe also pit latrine): IMO it would be more recognizable to show the "default" toilets symbol on the map
* specific kinds of fitness stations (maybe also playground features), see also https://github.com/openstreetmap/id-tagging-schema/pull/1163
